src/code/z_message_staff.c

```diff
@@ -35,12 +35,9 @@ f32 sCreditsFontWidths[144] = {
     14.0f, 14.0f, 14.0f, 14.0f, 14.0f, 14.0f, 14.0f, 14.0f, 14.0f, 14.0f, 14.0f, 14.0f, 14.0f, 14.0f, 14.0f, 14.0f,
 };
 
-#ifdef NON_MATCHING
-// https://decomp.me/scratch/yc8Or
-// v0/v1 swap between `character` and `i`
 void Message_DrawTextCredits(PlayState* play, Gfx** gfxP) {
     MessageContext* msgCtx = &play->msgCtx;
-    u16 lookAheadCharacter;
+    u16 pad;
```

Comment:
does s32 not work? its probably a `GameState`->`PlayState` cast.

Reply:
It does not unfortunately, it makes the stack too big no matter how you arrange it. And removing it messing up the stack location of the other variables.